src/parser/jobs/vpr/modules/SerpentsTail.tsx

```diff
@@ -74,9 +79,10 @@ export class SeprentsTail extends Analyser {
 	}
 
 	private onComplete() {
-		this.checklist.add(new Rule({
+		this.checklist.add(new TieredRule({
```

Comment:
I know in the past, Ack has made some noodling about moving away from TieredRules in the checklist, making things pass/fail.  Given the comments in the linked discord thread, could we do this with a Rule with `target: 99` or `target: 100` instead (using an appropriately named const value instead of a magic number, of course)?

Reply:
rolled it back to a rule and set a const target.

![image](https://github.com/user-attachments/assets/d34ac061-bd94-4699-98ca-64a5c51ae7ae)
![image](https://github.com/user-attachments/assets/a8eb92c0-b8ee-4149-b979-424eee9a6047)